### DISCUSSION TITLE
苹果手机ios版怎么不能用。

### DISCUSSION THREAD

**Discussion by ascia-z** [getlantern/lantern-forum-cn#657] [General]
买了很久了，苹果手机一只无法使用，这不是欺诈吗？？？

**Comment by MaxMKMax** [comment]
@ascia-z 您好. 有好消息了!!! 已经提交新版本给iOS商店发放了. 等通过. 应该这 3 天里面就能更新最新版本了.
您用不了专业版的时间我们会重新给您延长时间. 请您放心. 不会让您吃亏的. 等您更新好了之后, 再来找我帮您延长!
非常抱歉让您久等了! 谢谢您的耐心等待和支持我们!

**Comment by ascia-z** [comment]
目前苹果手机还是无法使用，希望你们实事求是，不要哄骗欺诈好吗？我还要再买个其他VPN专门在手机上使用？？

**Reply by MaxMKMax** [reply]
![image](https://github.com/getlantern/lantern-forum-cn/assets/155475142/d044348a-bd68-40ac-9fa2-658fbae56c70)
@ascia-z 您好. 没有哄骗的. 我们的app的功能无法通过iOS审核. 我们一直努力地重新提交了. 新版本都弄好了, 只是一直卡在iOS应用商店的批准. 我们一直在提交. IOS他们那里太严格了. 我们也头疼的.

**Reply by ascia-z** [reply]
那如何解决？

**Reply by ascia-z** [reply]
不可能凭你这个邮件就让消费者买单吧

**Comment by ascia-z** [comment]
现在苹果手机是否能用？

**Reply by MaxMKMax** [reply]
@ascia-z 我们又在和iOS的人开会讨论了. 我们这里会做修改来配合他们. 也只能这样了. IOS他们太刁钻了. 真的非常抱歉让您久等了! 我们也不想的. 一进提交上去好多次了. 一直被拒绝下来. 之后我们给您添加时间做补偿吧.
